conf/cmi/core.extension.yml

```diff
@@ -36,6 +36,9 @@ module:
   flysystem_azure: 0
   focal_point: 0
   gin_toolbar: 0
+  gnode: 0
```

Comment:
What is this?

Reply:
It's the Group Node module, which enables group functionality for nodes.